src/components/Calculator.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './calculatorStyle.scss';
+import CalculatorButton from './buttons/CalculatorBtn';
+import Display from './display/Display';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
```

Comment:
- [ ] Calculator js should have class based component using ES6 syntax. Please work on it.
```

'import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;

```

Reply:
Calculator already was a class

Reply:
- This issue is resolved. Thanks for making the changes requested by the previous reviewer. :clap: